test/README.md

```diff
@@ -13,7 +13,7 @@ There are two test suites associated with Mapbox GL JS
  To run individual tests:
 
  - Unit tests: `yarn test-unit path/to/file.test.js` where the path begins within the `/test/unit/` directory
- - Render tests: `yarn test-render render-test-name`
+ - Render tests: `yarn test-render render-test-name` (e.g. `yarn test-render background-color/default`)
```

Comment:
@mollymerp I added an example here because it wasn't entirely clear to me what the render-test-name is.